intro-to-tests/problem-set-intro-to-pytest.md

```diff
@@ -155,6 +171,20 @@ In tests, long test names are welcome. For each test case, the best test names u
 
 The following guideline is challenging to master, but the best test names tend to describe a situation, and not name exact variable names. If the variable name changes, then the test name could become inaccurate.
 
+Pytest will only recognize test names beginning with `test_` or ending with `_test`.
+
+##### !end-hint
+
+##### !explanation
+
+For testing the `calculate` function will return a `ValueError` when given an invalid `operator`, the best choice for the test name would be `test_calculate_invalid_operator_returns_error_msg()`. This is the best test name because one can read the name of the test and determine the conditions of the test.
```

Comment:
```suggestion
For testing that the `calculate` function will return a `ValueError` when given an invalid `operator`, the best choice for the test name would be `test_calculate_invalid_operator_returns_error_msg()`. This is the best test name because one can read the name of the test and determine the conditions of the test.
```